django_project/changes/templates/version/includes/version-list-detail.html

```diff
@@ -31,14 +31,16 @@ <h3>
                 </a>
                 {% endif %}
             {% endif %}
-            <a class="btn btn-default btn-mini"
-               href='{% url "version-delete" project_slug=version.project.slug slug=version.slug %}'>
-                <span class="glyphicon glyphicon-minus"></span>
-            </a>
-            <a class="btn btn-default btn-mini"
-               href='{% url "version-update" project_slug=version.project.slug slug=version.slug %}'>
-                <span class="glyphicon glyphicon-pencil"></span>
-            </a>
+            {% if version_owner %}
```

Comment:
maybe switch to user_can_delete and user_can_edit instead of adding a new variable version_owner

Reply:
Updated